const errores = {
    "error-evento":
        '<p>Hubo un error al intentar guardar el evento <br> Por favor, vuelva a intentarlo mas tarde</p>',
    "error-registro":
        '<p>Hubo un error al intentar guardar su registro <br> Por favor, vuelva a intentarlo mas tarde</p>',
    "error-bd":
        '<p>Hubo un error al intentar acceder a la base de datos <br> Por favor, vuelva a intentarlo mas tarde</p>',
    "mensaje-1":
        '</p>Intentá nuevamente! <br>Comprobá que la información este bien <br/> para continuar con el proceso. </p>',
    "error-invalido":
        '</p>Por motivos de seguridad debimos cerrar tu sesión. <br/>Inicia sesion nuevamente para continuar. </p>',
    "error-servidor":
        '<p>En este momento no es posible continuar por una falla del sistema. <br/> Te pedimos que intentes nuevamente para completar el proceso.</p>',
    "mensaje-2":
        '<p>Algo salio mal. <br/> Te pedimos que intentes nuevamente.</p>',
    "usuario-requerido":
        '<p>Para poder inscribirse al evento es necesario estar registrado. <br> Regístrate o Inicia sesión en caso de tener una cuenta.</p>',
    "usuario-duplicado":
        '<p>No se pudo completar el registro del usuario debido a que ya hay uno con el mismo DNI y email. <br> Intente con otros datos y vuelva a registrarse.</p>',
    };

export default errores;
